### ISSUE TITLE
Error logs are printed when the JSON report is generated. 

### ISSUE THREAD

**Issue by aabedraba** [daveshanley/vacuum#372]
**Issue:**

The following OpenAPI document contains references that are not resolved: https://pastebin.com/raw/9r1xKVu3

When running a spectral report against it, rolodex logs are printed with the report and the output is mixed. 

```sh
$ vacuum spectral-report -o /swagger.json

2023-11-22 14:32:15 ERROR unable to open rolodex file
                      ├ file: /Users/aabedraba/github/zuplo/rate-my-openapi/apps/api/networkInterface.json
                      └ error: rolodex has no file systems configured, cannot open '/Users/aabedraba/github/zuplo/rate-my-openapi/apps/api/networkInterface.json'. Add a BaseURL or BasePath to your configuration so the rolodex knows how to resolve references

[ # <-- report has been generated
    {
        "code": "openapi-tags",
        "path": [
            "$[0]"
        ],
        "message": "Top level spec `tags` must not be empty, and must be an array: `tags`, is missing and is required",
```

**Expected output**: 

- either no logs and just the JSON report (preferable in my case)
- error out completely and not generate the report

**Comment by daveshanley** [comment]
This should be fixed with `v0.5.1`

**Comment by aabedraba** [comment]
Tested and works as expected. Thanks so much!
